tests/integration/test_integration.py

```diff
@@ -783,4 +783,17 @@ def test_no_warnings():
     env["DD_TRACE_SQLITE3_ENABLED"] = "false"
     out, err, status, pid = call_program("ddtrace-run", sys.executable, "-Wall", "-c", "'import ddtrace'", env=env)
     assert out == b"", out
-    assert err == b"", err
+
+    # Wrapt is using features deprecated in Python 3.10
+    # See https://github.com/GrahamDumpleton/wrapt/issues/200
+    if sys.version_info < (3, 10, 0):
+        assert err == b"", err
+    else:
+        assert (
+            err
+            == (
+                b"<frozen importlib._bootstrap>:914: ImportWarning: "
+                b"ImportHookFinder.find_spec() not found; falling back to find_module()\n"
+            )
+            * 75
+        ), err
```

Comment:
hmm... I wonder what we should do here... feels weird to allow this kind of error to be raised.

Is there a change we can make to wrapt?

Reply:
> This was complicated slightly by a deprecation warning new in Python 3.10 as a result of some features wrapt is using which are slated for removal soon. An issue was created upstream to address since the fix didn't appear to be trivial.

I gave it a shot but wasn't near close to confident in the fix. Obviously not ideal but don't think we should be doing anything further at the moment.